### PR TITLE
KOGITO-5069 Bump Quarkus 2.0.0.CR3

### PR DESCRIPTION
**Pull request by evacchi**
- https://github.com/kiegroup/kogito-runtimes/pull/1274
- https://github.com/kiegroup/kogito-apps/pull/857
- https://github.com/kiegroup/kogito-examples/pull/722
- https://github.com/kiegroup/optaplanner/pull/1379

### CODE REVIEW

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by danielezonca** [comment]
I see in the logs many WARNING messages like
`[WARNING] [io.smallrye.openapi.runtime.scanner] SROAP04005: Could not find schema class in index: java.lang.Object`
That is not in logs based on Quarkus 1.x version

It looks like something related to SmallRye openApi but I don't have any idea.

@evacchi
Do you know who can be the best fit from quarkus team to have a look?

**Comment by evacchi** [comment]
> I see in the logs many WARNING messages like
> `[WARNING] [io.smallrye.openapi.runtime.scanner] SROAP04005: Could not find schema class in index: java.lang.Object`
> That is not in logs based on Quarkus 1.x version
> 
> It looks like something related to SmallRye openApi but I don't have any idea.

@phillip-kruger is that alright?

**Comment by phillip-kruger** [comment]
mmm, not sure. @MikeEdgar might be able to comment ?

**Comment by MikeEdgar** [comment]
@phillip-kruger, seems odd... did something change in v2 with how Quarkus (or the OpenAPI extension) builds the index?

**Comment by phillip-kruger** [comment]
Not sure. Nothing that I know about. But might be somewhere in core ?

**Comment by evacchi** [comment]
jenkins please retest this

File: README.md [hunk 1]
```diff
@@ -108,3 +108,4 @@ shows timers (intermediate and boundary) that allows to introduce delays in proc
 Every example has a directory named `operator` including the YAML files to deploy it using the Kogito Operator in an OpenShift cluster.
 Please refer to the [Kogito Documentation](https://docs.jboss.org/kogito/release/latest/html_single/#chap_kogito-deploying-on-openshift)
 of how to install the operator to your environment in order to try it there.
+
```
Comment (danielezonca) [inline README.md]: to be removed

**Review by danielezonca — APPROVED** [review]
Approved, just a minor comment

Btw I would like to have feedback from SmallRye OpenApi team about the warnings

**Review by cristianonicolai — APPROVED** [review]

**Comment by cristianonicolai** [comment]
Jenkins rerun examples tests

**Comment by oblodgett** [comment]
> > I see in the logs many WARNING messages like
> > `[WARNING] [io.smallrye.openapi.runtime.scanner] SROAP04005: Could not find schema class in index: java.lang.Object`
> > That is not in logs based on Quarkus 1.x version
> > It looks like something related to SmallRye openApi but I don't have any idea.

I was getting this warning and my interface looked like:

```
@Produces(MediaType.APPLICATION_JSON)
@Consumes(MediaType.APPLICATION_JSON)
public interface BaseCrudRESTInterface<E> {

	@POST
	@Path("/")
	public E create(E entity);
```

Changing `BaseCrudRESTInterface<E>` -> `BaseCrudRESTInterface<E extends BaseEntity>` makes the warning go away.

**Pull request merged**